mobile_app_connector/models/firebase_notification.py

```diff
@@ -36,6 +36,9 @@ class FirebaseNotification(models.Model):
     fundType = fields.Many2one("product.product", "Fund product", readonly=False)
     child_id = fields.Many2one("compassion.child", "Child", readonly=False)
 
+    res_model = fields.Many2one("ir.model")
```

Comment:
I  think it's more simple to store directly the name of the related model instead of the id in the table `ir.model`
```suggestion
    res_model = fields.Char()
```